web/client/plugins/StyleEditor.jsx

```diff
@@ -132,7 +132,7 @@ const StyleEditorPlugin = compose(
                 isEditing: status === 'edit',
                 loading,
                 layer,
-                error: !!(error && error.availableStyles),
+                error: !!(error?.availableStyles || error?.global),
```

Comment:
We should specify the error in the message. Currently the title is always 'Missing available styles' and we are adding the error.global as additional flag.

As discussed with @MV88 we should pass the whole error object to the emptyState enhancer and change the content of the message description based on the type of error, something like this:

```js
emptyState(
    ({ error }) => !!(error?.availableStyles || error?.global),
    ({ error }) => {
        if (error.availableStyles) {
            return {....};
        }
         return {....};
    }
),
 
```
we should catch only `error?.availableStyles || error?.global`